Classes/BounceHandler.php

```diff
@@ -102,7 +102,6 @@ protected function findEmail()
 
```

Comment:
I believe those blank lines, right before a comment, add to the readability by visually grouping things (and separating from the following "block"). I won't merge this commit.